library/nist_kw.c

```diff
@@ -66,7 +66,13 @@ static inline unsigned char mbedtls_nist_kw_safer_memcmp( const void *a, const v
     volatile unsigned char diff = 0;
 
     for( i = 0; i < n; i++ )
-        diff |= A[i] ^ B[i];
+    {
+        /* Read volatile data in order before computing diff.
+         * This avoids IAR compiler warning:
+         * 'the order of volatile accesses is undefined ..' */
```

Comment:
That's a good warning.

Reply:
copy \ pasted from the original in `ssl_internal.h`
Created https://github.com/ARMmbed/mbedtls/issues/1857 for unifying them to a single module

Reply:
Thanks